brave/brave-core
# Fix message signature for Trezor accounts

Resolves https://github.com/brave/brave-browser/issues/19504

- Fix message signature for Trezor accounts
- Removed types duplication

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

   1. Connect Trezor 
   2. Visit cryptokitties.co and try to sign mesage via trezor wallet
   3. Sign message popup goes into loop 
   4. Check signature is success, you have to be logged in to cryptokitties
